packages/gatsby-theme-docs/src/utils/helpers/generic.js

```diff
@@ -0,0 +1,40 @@
+import unified from 'unified';
+import markdown from 'remark-parse';
+import remark2react from 'remark-react';
+
+export const markdown2React = markdownString => {
+  const result = unified()
+    .use(markdown, { commonmark: true })
+    .use(remark2react)
+    .processSync(markdownString).contents;
+
+  return result;
+};
+
+export const computeType = value => {
+  const regex = /^\[object (\S+?)\]$/;
+  const matches = Object.prototype.toString.call(value).match(regex) || [];
+
+  return (matches[1] || 'undefined').toLowerCase();
+};
+
+export const generateAnchorString = str => {
```

Comment:
Same here, where is this supposed to be used?

Reply:
Used to generate an ID for the page title.

The right/index nav's title is expected to be clickable and the page view should jump to the title - does this make sense?

Reply:
Sure but this already works, so I'm not sure where this function is supposed to be used 🤔